ex04.ex

```diff
@@ -54,8 +55,17 @@ defmodule Ex04 do
       -7
 
   """
+  def getSmaller(a,b), do:
+	if a > b do: 
+		a
+	else 
+		b
+	end
+  end
 
-  def min . . . "your code"
+  def min([list]), do: 
```

Comment:
## -3

I'm giving you a little benefit of the doubt here.

In theory, the code submitted uses the library function `min/2`, which isn't permitted. But you wrote your own implemented, and I'm guessing you just forgot to use it.

But... I'm not keen on the use of`if` in that function.

~~~ elixir
  def min(list), do: reduce(list, &min_of/2)
  def min_of(a,  b) when a < b, do: a
  def min_of(_a, b), do: b
~~~